src/constants/java/net/fabricmc/yarn/constants/CatTypes.java

```diff
@@ -0,0 +1,18 @@
+package net.fabricmc.yarn.constants;
```

Comment:
Should these be put into their own package?

Reply:
Might be something for another PR to move the existing ones around, so dont worry about it too much for now.